jdat_notebooks/redshift_crosscorr/requirements

```diff
@@ -0,0 +1,2 @@
+specutils==1.0
+astropy==4.0
```

Comment:
Instead of pinning dependencies in `requirements.txt` which prevents the notebook from benefitting from bug fixes, wouldn't it be better to to have the specifier indicate minimum versions of these packages needed?  I.e. something like:

```
specutils>=1.0
astropy>=4.0
```

or whatever the actual minimum versions are for this notebook (which are unlikely to be the latest above).

Reply:
Good question. I have noticed a few times that a new version of a package can break the caller's code. It sounds like a trade-off in which reproducibility plays a part. I wonder if reproducibility should be emphasized at the expense of other factors. Since, *for the one particular notebook*, any underlying bugs should not be affecting the results.